readthedocs/projects/models.py

```diff
@@ -306,6 +306,11 @@ class Project(models.Model):
         default=False,
         help_text='If checked, do not show advertising for this project',
     )
+    is_spam = models.NullBooleanField(
```

Comment:
In https://github.com/readthedocs/readthedocs.org/pull/8717/files#diff-2026582e018afc479271c2c856ebf5163912b778923ab381526f03d3aa2363a3 you are replacing NullBoolean fields with a boolean field that accepts null values, probably better to use a boolean field here.

Reply:
Agreed, this seems useful 👍